code/_onclick/item_attack.dm

```diff
@@ -81,7 +81,7 @@
 	M.lastattackerckey = user.ckey
 
 	user.do_attack_animation(M)
-	. = M.attacked_by(src, user, def_zone)
+	. = !M.attacked_by(src, user, def_zone)
```

Comment:
This is kind of a deep change, have you verified that this doesn't have any other implications with attacks?

Reply:
as i stated above, it only denies `afterattack`, nothing else
from what i've seen, there are not many weapons with afterattack and nothing should be broken

Reply:
no, i did not test everything with afterattack, but i tested attacking with cult dagger and blade against someone parrying and not parrying and it worked as i wanted